Marlin/src/pins/sanguino/pins_ANET_10.h

```diff
@@ -89,6 +89,12 @@
  *   Many thanks to Hans Raaf (@oderwat) for developing the Anet-specific software and supporting the Anet community.
  */
 
+#ifdef TARGET_LPC1768
+  #error "Oops! Set MOTHERBOARD to an LPC1768-based board when building for LPC1768."
+#elif defined(__STM32F1__)
+  #error "Oops! Set MOTHERBOARD to an STM32F1-based board when building for STM32F1."
+#endif
```

Comment:
The logic of these tests is wrong. What you want is something like this:
```cpp
#if !defined(TARGET_LPC1768) && !defined(__STM32F1__)
  #error "Oops! Set MOTHERBOARD to an LPC1768 or STM32F1-based board when building Sanguino Anet 10."
#endif
```
Of course, this test also makes no sense. This pins file wouldn't be getting included and compiled if the `MOTHERBOARD` value didn't require it. So I think this PR is going to be closed until you figure out what you're trying to do.